pkg/build/api/types.go

```diff
@@ -164,6 +164,12 @@ type GitBuildSource struct {
 
 	// Ref is the branch/tag/ref to build.
 	Ref string
+
+	// HTTPProxy is a proxy used to reach the git repository over http
+	HTTPProxy string `json:"httpProxy,omitempty"`
```

Comment:
yeah but we haven't removed it all yet either so i'm going for consistency :)

i'll remove it.